azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/VirtualMachineScaleSet.java

```diff
@@ -221,6 +222,19 @@
     Map<String, VirtualMachineScaleSetExtension> extensions();
 
     /**
+     * @return the network interfaces associated with all virtual machine instances in a scale set
+     */
+    PagedList<VirtualMachineScaleSetNetworkInterface> listNetworkInterfaces();
```

Comment:
Why specifically PagedList<T>? Why not simply List<T>?  I believe everywhere else we return List<T> we return List<T>, and PageList<T> is just an internal impl detail?